.github/workflows/openms_ci_matrix_full.yml

```
@@ -327,16 +327,21 @@
           cd ..
           # add third-party binaries to PATH
           # use flat THIRDPARTY structure
+          $THIRDPARTY_DIR = "OpenMS/THIRDPARTY"
           mkdir -p _thirdparty
           # Copy the correct architecture
-          cp -R OpenMS/THIRDPARTY/${{ steps.set-vars.outputs.tp_folder }}/`uname -m`/* _thirdparty/
-          # For MacOS 14 we also want to copy over any tools that we only have in x86_64 flavor. cp -n avoids overwriting the arm specific executables that we've already copied above
+          cp -R $THIRDPARTY_DIR/${{ steps.set-vars.outputs.tp_folder }}/`uname -m`/* _thirdparty/
+          
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)